eng/Version.Details.xml

```diff
@@ -49,6 +49,50 @@
       <Uri>https://github.com/dotnet/runtime</Uri>
       <Sha>5535e31a712343a63f5d7d796cd874e563e5ac14</Sha>
     </Dependency>
+    <Dependency Name="System.Composition" Version="8.0.0">
```

Comment:
@dotnet/source-build-internal should this have `Pinned=true` on it? These are new entries. cc @mmitche

Reply:
This should get pinned I think, yes.